http-gateway/web/src/common/utils/parse-streamed-data.js

```diff
@@ -0,0 +1,11 @@
+// Return an array of objects which came in as a text stream of results
+export const parseStreamedData = stream =>
+  stream
+    ? stream
+        .trim()
+        .split('\n\n')

```

Comment:
Messages are separated by\n\n?

Reply:
They are, it's a long story, but in short, the stream sends you binary data, decoded to text, which then must be converted to JSON, that's why the `\n\n`, cause its separated with it.